tests/common/compact_test.go

```diff
@@ -46,11 +46,7 @@ func TestCompact(t *testing.T) {
 			defer clus.Close()
 			testutils.ExecuteWithTimeout(t, 10*time.Second, func() {
 				var kvs = []testutils.KV{{Key: "key", Val: "val1"}, {Key: "key", Val: "val2"}, {Key: "key", Val: "val3"}}
-				for i := range kvs {
-					if err := clus.Client().Put(kvs[i].Key, kvs[i].Val); err != nil {
-						t.Fatalf("compactTest #%d: put kv error (%v)", i, err)
-					}
-				}
+				maintenanceInitKeys(t, clus.Client(), kvs...)
```

Comment:
I have reservations about creating a helper function here. I think as long this is a single loop with calling put, it's more clearer than function call. Proposed name `maintenanceInitKeys` doesn't really make it obvious what it does. We could fix the name, however I would argue that this code is simple enough that it's not worth to do so.

Can you just revert this change?

Reply:
Ok.